src/js/components/actionlist/index.js

```diff
@@ -69,9 +69,6 @@ export default Picklist.extend({
   onSelect({ model }) {
     if (model.get('isDisabled')) return;
 
-    const onSelect = model.get('onSelect');
```

Comment:
Ah we definitely need to call `onSelect`, we just don't need to check if `onSelect` exists prior to calling it.